apps/web/vibes/soul/components/cart/index.tsx

```diff
@@ -1,230 +1,226 @@
-                              <div className="grid w-full grid-cols-1 gap-5 @sm:col-span-2 @sm:mt-10 @md:grid-cols-2">
-                                <Button variant="secondary" className="order-2 w-full @md:order-1">
-                                  Cancel
-                                </Button>
-                                {/* TODO: disbale until form is complete */}
-                                <Button disabled variant="primary" className="w-full">
-                                  Add Address
-                                </Button>
-                              </div>
-                            </form>
-                          </div>
-                        }
-                      />
-                    </td>
-                  </tr>
-                  <tr className="border-b border-contrast-100">
-                    <td>Tax</td>
-                    <td className="py-4 text-right">$4.50</td>
-                  </tr>
-                </tbody>
-                <tfoot>
+        <div className="@4xl:w-1/3">
+          <h2 className="mb-10 font-heading text-4xl font-medium leading-none @xl:text-5xl">
+            {summary.title ?? 'Summary'}
+          </h2>
+          <table aria-label="Receipt Summary" className="w-full">
+            <caption className="sr-only">{summary.caption ?? 'Receipt Summary'}</caption>
+            <tbody>
+              <tr className="border-b border-contrast-100">
+                <td>{summary.subtotalLabel ?? 'Subtotal'}</td>
+                <td className="py-4 text-right">{summary.subtotal}</td>
+              </tr>
+              <tr className="border-b border-contrast-100">
+                <td>{summary.shippingLabel ?? 'Shipping'}</td>
+                <td className="py-4 text-right">TBD</td>
+              </tr>
+              <tr>
+                <td>Tax</td>
+                <td className="py-4 text-right">TBD</td>
+              </tr>
+            </tbody>
+            {/* TODO: when shipping and tax are TBD, it doesn’t make sense to display Grand Total here... Commenting out for now. Should I remove it? In that case should we remove the summary.grandTotalLabel & summary.grandTotal prop defs? */}
+            {/* <tfoot>
                   <tr className="text-xl">
                     <th scope="row" className="text-left">
-                      Grand Total
+                      {summary.grandTotalLabel ?? 'Grand Total'}
                     </th>
-                    <td className="py-10 text-right">$59.50</td>
+                    <td className="py-10 text-right">{summary.grandTotal}</td>
                   </tr>
-                </tfoot>
-              </table>
-              <Button className="w-full">Checkout</Button>
+                </tfoot> */}
+          </table>
+          <Button className="mt-10 w-full">{summary.ctaLabel ?? 'Checkout'}</Button>
+          {/* TODO: Is this supposed to be a form submission as well? with the redirectToCheckoutAction prop passed? */}
```

Comment:
Yes, you need to add a `form` and use this button as submit. Call `redirectToCheckoutAction` as you mention.